XXX: pkg-config bug:
Package gsettings-desktop-schemas>=40.alpha was not found in the pkg-config search path

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -23,7 +23,7 @@ uprof_req = '>= 0.3'
 pango_req = '>= 1.46.0'
 cairo_req = '>= 1.10.0'
 pangocairo_req = '>= 1.20'
-gsettings_desktop_schemas_req = '>= 40.alpha'
+gsettings_desktop_schemas_req = '>= 40'
 json_glib_req = '>= 0.12.0'
 xcomposite_req = '>= 0.4'
 xkbcommon_req = '>= 0.4.3'
